assets/stylesheets/_mixins.scss

```diff
@@ -419,8 +419,12 @@
 
 		/* Fonts smaller than 16px causes mobile safari to zoom. */
 		font-size: $mobile-text-min-font-size;
+		/* Override core line-height. To be reviewed. */
```

Comment:
What's this "To be reviewed" comment? Should that be removed?

Reply:
In the next future we will need to set an explicit value for line-height. Letting it to the browsers default is not ideal now that we use `min-height` on the input fields instead of a fixed height. Note that in Gutenberg some fields still use a fixed height. Once we change that to `min-height` we'll start seeing issues in Firefox :)

Reply:
Ok, thanks. Can you expand this comment to make that a little more clear?